src/components/articles/ArticlePage.jsx

```diff
@@ -40,8 +42,11 @@ const ArticlePage = () => {
         <p className='paragraph'>{article.body}</p>
         <span className='comments'>Comments on this article: {article.comment_count}</span>
       </div>
+
+      <div id='comments'>
```

Comment:
Some of the `div`s feel unnecessary - the styling can be applied to the `ul` that the `Comments` component returns instead.